distributed/asyncio.py

```diff
@@ -0,0 +1,61 @@
+import asyncio
```

Comment:
It would be nice if we could avoid the import cost of `asyncio` for people who don't use it.